README.md

```
@@ -45,38 +45,99 @@
 ``` sh
 
 $ npx create-cljs-project your-project
-$ npm install react autoprefixer babel-loader css-loader cssnano karma karma-chrome-launcher karma-cljs-test msw npm-run-all postcss postcss-cli postcss-loade postcss-preset-env shadow-cljs tailwindcss webpack webpack-cli rimraf glob @isaacs/cliui --save-dev
+$ npm install react react-dom react-refresh autoprefixer css-loader cssnano karma karma-chrome-launcher karma-cljs-test msw npm-run-all postcss postcss-cli postcss-loader postcss-preset-env shadow-cljs tailwindcss rimraf glob cliui --save-dev
```

Markdownlint

Dollar signs used before commands without showing output

(MD014, commands-show-output)